docs/content/recipes/interoperability/identifier-mapping.md

```diff
@@ -0,0 +1,384 @@
+to dynamically select which mappings to consider relevant and which to ignore. For example allowing or disallowing mappings between stereoisomers or between genes and proteins.
+
+Examples of types of mappings are:
+* **Content mapping**: where we are mapping the actual entities by using techniques such as BLAST in biological sequences or comparison of INCHI identifiers for chemical compounds
+* **Ontology mapping**: this can either be 
+    * As a direct 1-to-1 mapping between equivalent terms in different ontologies
+    * As a complex m-to-m mapping between terms in different ontologies taking into account their hierarchical structure, see [[2]](#References).
+* **Identifier mapping**: The focus of this relationship. This can either be:
+    * Mapping between differently formed identifiers that resolve to the same entity.
+    * Mapping between identical local identifiers with different namespaces
+    * Mapping between entities that are related enough to be usefully connected (e.g. linking information on proteins, genes, RNA and reporter sequences for these)
+    * Mapping between databases containing different information about the same entity (e.g. links between the protein sequence database UniProt and the protein 3D structure database PDB)
+
+
+___
+
+## Identifier Mappings
+
+> :notebook_with_decorative_cover: **"Identifiers are used to tag, identify, find and retrieve entities which are part of a collection or a resource maintained by some organization. "**
+
+To satisfy the Findability criteria F1, organisations must create identifiers for the concepts within their database. This generates locally unique identifiers which can in turn be transformed into globally unique identifiers using namespaces for which the database organisation are the `authority`. 
+
+Databases often contain data that exists in, or is closely related to, the content of other databases. For example, the genome database [Ensembl](http://ensembl.org/) contains data about genes that are related to entries in databases such as HUGO Gene Nomenclature Committee (https://www.genenames.org/) or [NCBI Gene](https://www.ncbi.nlm.nih.gov/gene/); or a database about drugs, e.g. [DrugBank](https://drugbank.com/), often contains details of the chemical substance that forms the drug which are also contained in chemical databases such as [ChEBI](https://www.ebi.ac.uk/chebi/) or [PubChem](https://pubchem.ncbi.nlm.nih.gov/). This results in a large number of unique identifiers notionally for the same concept. This large number of identifiers for the same concept prevents interoperability of the data, since additional knowledge is needed to know which identifiers represent the same concept from different databases. However, the databases often contain `cross-reference` links to other databases to represent these `equivalences`.
```

Comment:
Consider adding proper literature citations in addition to the URLs.

Reply:
Alasdair added these links so not sure whether I should change it